_includes/gs3-sponsors.html

```diff
@@ -47,6 +52,11 @@
        title="HL Consulting"
        style="display:inline; margin-left:5px; margin-right:5px; min-width: 80px; max-width: 150px; max-height: 80px;"/> 
 </a>
+<a href="http://ign.fr/">
+  <img src="/sponsor/img/ign-logo.png"
+       title="National Institute of Geographic and Forest Information (IGN)"
```

Comment:
Based on [wikipedia](https://en.wikipedia.org/wiki/Institut_national_de_l%27information_géographique_et_forestière) this could be "Institut national de l'information géographique et forestière (IGN)"